### PR TITLE
vlad-fix-part3

### PR DESCRIPTION
**Pull request by vlad666spot**
Fixed the next API calls:
- AWS EG 
- AWS EG EMR

### CODE REVIEW

File: api/services/elastigroup/azure/scaleSetVms/paths/groups.yaml [hunk 1]
```diff
@@ -28,30 +28,7 @@ post:
       application/json:
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/scaleSetVms/paths/groups.yaml:28]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/scaleSetVms/schemas/lowPriorityVMsEGCreation.yaml [hunk 2]
```diff
@@ -0,0 +1,1202 @@
+type: "object"
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/scaleSetVms/schemas/lowPriorityVMsEGCreation.yaml:1]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

**Review by sagishoshan — CHANGES_REQUESTED** [review]

File: api/services/elastigroup/aws/paths/instance-lock.yaml [hunk 3]
```diff
@@ -8,14 +8,6 @@ post:
   parameters:
     - $ref: "../../aws/parameters/instanceId.yaml"
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - in: "query"
-      name: "ttl"
```
Comment (sagishoshan) [inline api/services/elastigroup/aws/paths/instance-lock.yaml:12]: @vlad666spot why are you removing this query param?
the name should be fixed to 'ttlInMinutes' like in the old documentation -http://help.dev.spotinst.com/spotinst-api/elastigroup/amazon-web-services/lock-instances/

File: api/services/elastigroup/aws/paths/logs.yaml [hunk 4]
```diff
@@ -27,9 +27,15 @@ get:
     - $ref: "../../../../commons/parameters/accountId.yaml"
     - in: "query"
       name: "severity"
-      example: "all"
-      description: >
-        Filter log extracted entires related to a specific resource id
+      example: "ALL"
+      description: "Filter log extracted entires related to a specific resource id </br> Valid values:
```
Comment (sagishoshan) [inline api/services/elastigroup/aws/paths/logs.yaml:31]: @vlad666spot lets use enum

File: api/services/elastigroup/azure/scaleSetVms/paths/group-deployments.yaml [hunk 5]
```diff
@@ -6,8 +6,31 @@ get:
   tags:
     - Elastigroup Azure Scale Set VMs
   parameters:
-    - $ref: "../../../../../commons/parameters/accountId.yaml"
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/scaleSetVms/paths/group-deployments.yaml:9]: @vlad666spot this part is already addressed in the master, please make sure you are aligned (no need to re-update)

File: api/services/elastigroup/azure/scaleSetVms/paths/roll.yaml [hunk 6]
```diff
@@ -0,0 +1,65 @@
+get:
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/scaleSetVms/paths/roll.yaml:1]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/scaleSetVms/responses/roll.yaml [hunk 7]
```diff
@@ -0,0 +1,72 @@
+description: Roll Status Response
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/scaleSetVms/responses/roll.yaml:1]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/scaleSetVms/responses/startDeployment.yaml [hunk 8]
```diff
@@ -0,0 +1,72 @@
+description: Start Deployment Response
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/scaleSetVms/responses/startDeployment.yaml:1]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/scaleSetVms/schemas/roll.yaml [hunk 9]
```diff
@@ -0,0 +1,23 @@
+type: object
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/scaleSetVms/schemas/roll.yaml:1]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/scaleSetVms/schemas/startDeployment.yaml [hunk 10]
```diff
@@ -0,0 +1,36 @@
+type: object
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/scaleSetVms/schemas/startDeployment.yaml:1]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/spotVms/paths/groupVm.yaml [hunk 11]
```diff
@@ -7,6 +7,14 @@ put:
     - Elastigroup Azure Spot VMs
   parameters:
     - $ref: "../../../../../commons/parameters/accountId.yaml"
+    - in: "path"
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/spotVms/paths/groupVm.yaml:10]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/spotVms/paths/groupVmDeploymentDetails.yaml [hunk 12]
```diff
@@ -8,7 +8,7 @@ get:
   parameters:
     - $ref: "../../../../../commons/parameters/accountId.yaml"
     - $ref: "../../../../../commons/parameters/groupId.yaml"
-    - in: query
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/spotVms/paths/groupVmDeploymentDetails.yaml:11]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/spotVms/paths/groupsVm.yaml [hunk 13]
```diff
@@ -13,7 +13,7 @@ post:
       application/json:
         schema:
           allOf:
-          - $ref: "../schemas/elastigroupVm.yaml"
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/spotVms/paths/groupsVm.yaml:16]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/spotVms/paths/vmSignal.yaml [hunk 14]
```diff
@@ -1,7 +1,7 @@
 post:
   summary: "Create VM Signal"
   description:
-    CThe VM signal API is used for notifying Spot about the VM state so that Spot can act accordingly.
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/spotVms/paths/vmSignal.yaml:4]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/spotVms/paths/vmLogs.yaml [hunk 15]
```diff
@@ -10,15 +10,15 @@ get:
     - $ref: "../../../../../commons/parameters/accountId.yaml"
     - $ref: "../../../../../commons/parameters/groupId.yaml"
     - in: query
-      name: FROM_DATE
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/spotVms/paths/vmLogs.yaml:13]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

File: api/services/elastigroup/azure/spotVms/paths/groupVmDeployment.yaml [hunk 16]
```diff
@@ -8,7 +8,7 @@ get:
   parameters:
     - $ref: "../../../../../commons/parameters/accountId.yaml"
     - $ref: "../../../../../commons/parameters/groupId.yaml"
-    - in: query
```
Comment (sagishoshan) [inline api/services/elastigroup/azure/spotVms/paths/groupVmDeployment.yaml:11]: @vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master

**Review by sagishoshan — CHANGES_REQUESTED** [review]

File: api/services/elastigroup/aws/paths/instance-lock.yaml (hunk 3 above)
Comment (vlad666spot) [reply api/services/elastigroup/aws/paths/instance-lock.yaml:12]: Screenshot 2020-11-04 at 15 43 48
fixed

**Review by sagishoshan — APPROVED** [review]

**Pull request merged**